### DISCUSSION TITLE
[http client] can't uncompress content starting with: ï»¿xœ«VJ­(È,J,ÉÌÏS²R22QÒQ*.I,)-òò³•j¾í Ð | HELP ME

### DISCUSSION THREAD

**Discussion by analuiza2102** [glpi-project/glpi-agent#751] [Q&A]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

Good morning, good afternoon, good night everyone!!

Could you help me?

A few weeks ago I updated the company's GLPI and updated the GLPI inventory together, we used fusion.

The inventory worked for a few days without giving an error, but now it is giving the following error:

```
[Mon Sep 2 09:09:54 2024][info] sending prolog request to server0
[Mon Sep 2 09:09:54 2024][error] [http client] can't uncompress content starting with: ï»¿xœ«VJ(È,J,ÉÌÏS²R22QÒQ*.I,)- òò³•j ¾í
Ð
```

Could you help me please?

**Comment by g-bougard** [comment]
Hi @analuiza2102 

this means your glpi server is not more answering any expected request. If it works before, this means something changed somewhere on the way, probably something was changed on your GLPI httpd server configuration.

Of course, you should also check the url configured on your agent is correct.

**Comment by analuiza2102** [comment]
Hi @g-bougard 

I updated the Inventory today, for context. My GLPI INVENTORY was downloaded via marktplace and the agent I downloaded the executable from this link:
https://github.com/glpi-project/glpi-agent/releases

At the beginning we had done a manual installation and downloaded it via marktplace, today the uninstallation was done, I reinstalled the inventory, I applied the correction to the Agent and the error remained the same. The path I'm using in the target is:

http://ip_do_meu_servidor/marketplace/glpiinventory/

Is this path correct?

Question 2:

The error you are pointing out >
[Fri Sep 6 11:58:14 2024][error] No supported answer from server at http://ip_do_meu_server/marketplace/glpiinventory/
[Fri Sep 6 11:58:14 2024][info] target server0: next run: Fri Sep 6 12:06:14 2024 - http://ip_do_meu_server/marketplace/glpiinventory/
[Fri Sep 6 11:58:14 2024][info] GLPI Agent memory usage: WSS=3227648 PFU=141828096
[Fri Sep 6 12:06:14 2024][info] target server0: server http://ip_do_meu_server/marketplace/glpiinventory/
[Fri Sep 6 12:06:14 2024][info] sending prolog request to server0
[Fri Sep 6 12:06:14 2024][error] [http client] can't uncompress content starting with: ï»¿xœ«VJ(È,J,ÉÌÏS²R22QÒQ*.I,)- òò³•j ¾í

**Reply by g-bougard** [reply]
> Hi @g-bougard
> 
> I updated the Inventory today, for context. My GLPI INVENTORY was downloaded via marktplace and the agent I downloaded the executable from this link: https://github.com/glpi-project/glpi-agent/releases
> 
> At the beginning we had done a manual installation and downloaded it via marktplace, today the uninstallation was done, I reinstalled the inventory, I applied the correction to the Agent and the error remained the same. The path I'm using in the target is:
> 
> http://ip_do_meu_servidor/marketplace/glpiinventory/
> 
> Is this path correct?
> 
> Question 2:
> 
> The error you are pointing out > [Fri Sep 6 11:58:14 2024][error] No supported answer from server at http://ip_do_meu_server/marketplace/glpiinventory/ [Fri Sep 6 11:58:14 2024][info] target server0: next run: Fri Sep 6 12:06:14 2024 - http://ip_do_meu_server/marketplace/glpiinventory/ [Fri Sep 6 11:58:14 2024][info] GLPI Agent memory usage: WSS=3227648 PFU=141828096 [Fri Sep 6 12:06:14 2024][info] target server0: server http://ip_do_meu_server/marketplace/glpiinventory/ [Fri Sep 6 12:06:14 2024][info] sending prolog request to server0 [Fri Sep 6 12:06:14 2024][error] [http client] can't uncompress content starting with: ï»¿xœ«VJ(È,J,ÉÌÏS²R22QÒQ*.I,)- òò³•j ¾í

Hi @analuiza2102 

the path seems correct, but the error, "No supported answer from server", also seems tell us this is not the case.

To me, your http server is not servicing requests as expected. You should double-check how you configured your GLPI server configuration.

On agent-side, you may enable debug=2 in registry and restart the service to see more detailed error and maybe this will give you some clue to understand what can be wrong server side.
